docs/instance.md

```diff
@@ -11,16 +21,42 @@ An instance is created on the command line:
 ```sh
 $ cozy-stack instances add <domain>
 ```
+With some possible additional options
+
+- `--locale <lang>`
+- `--email <email>`
+- `--environment <dev/test/production>`
+- `--apps <app1,app2,app3>`
+- `--home <cozy-home>`
+- `--onboarding <cozy-onboarding>`
+- `--registry https://registry.cozycloud.cc`
+
+It registers the instance in a global couchdb database `global/instances`
+```json
+{
+    "hostname": "example.cozycloud.cc",
+    "dbprefix": "example-clozycloud-cc/",
+    "fsroot": "/var/lib/cozy/example.cozycloud.cc/fs/"
+
+}
+```
 
-It registers the instance in a global database (**TODO** which one), and
-creates some databases for these doctypes:
+and creates the proper databases ($PREFIX/$DOCTYPE) for these doctypes:
 
 - `io.cozy.apps`
 - `io.cozy.manifests`
 - `io.cozy.files`
+- `io.cozy.folders`
```

Comment:
[future] I think that we will merge the files and folders inside the same database, so that we can send several documents to update in the same request (`_bulk_docs`)